Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -924,6 +925,12 @@ extension HTTPClient {
         /// The `Logger` used by the `Task` for logging.
         public let logger: Logger  // We are okay to store the logger here because a Task is for only one request.
 
+        @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+        public var tracer: (any Tracer)? {
```

Comment:
Same re public - does it have to be?

Reply:
I kinda think it's the same as Logger; if Logger is public this would be too